src/main/scala/mesosphere/marathon/api/v2/AppTasksResource.scala

```diff
@@ -126,7 +136,7 @@ class AppTasksResource @Inject() (
       val deploymentF = taskKiller.killAndScale(pathId, findToKill, force)
       deploymentResult(result(deploymentF))
     } else {
-      reqToResponse(taskKiller.kill(pathId, findToKill, wipe)) {
+      reqToResponse(taskKiller.kill(pathId, findToKill, wipe, -1)) {
```

Comment:
For some reason, not explicitly passing the -1 argument, even though its a default causes the mockito `verifyNoMoreInteractions(taskKiller)` check to fail. I didn't look into it too much but I believe it has to do with the way scala implements default arguments internally.